Submissions.php

```diff
@@ -528,7 +528,10 @@ function updatev(id)
                                                                                                                                                                                                              title:'Update Report Visibility',
                                                                                                                                                                                                              buttons: {
             'Update': function () {
+
                 $('#submit-form').submit()
+                $('#submit-form').submit();
```

Comment:
@enockkays  What is the point of having duplicate lines at line 532 and 533?  The line "$('#submit-form').submit()" appears twice at the two places.

-Hui